docs/tutorial.rst

```diff
@@ -0,0 +1,1613 @@
+provides the fba function in the ``psamm-model`` command to perform FBA on metabolic
+models. For example, to run FBA on the E. coli core model first make sure that
+the current directory is the E_coli_yaml/ directory using the following command:
+
+.. code-block:: shell
+
+    (psamm-env) $ cd <PATH>/psamm-tutorial/E_coli_yaml/
+
+Then run FBA on the model with the following command.
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-model fba
+
+Note that the command above should be executed within the folder that stores
+the model.yaml file. Alternatively, you could run the following command anywhere
+in your file system:
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-model --model <PATH-TO-MODEL.YAML> fba
+
+By default, `PSAMM` fba will use the biomass function designated in the central
+model file as the objective function. If the biomass tag is not defined in a
+model.yaml file or if the user wants to use a different reaction as the
+objective function, they can simply specify what the reaction ID is at the
+end of the command. For example to maximize the citrate synthase reactions,
+R_CS, the command would be as follows:
+
+.. code-block:: shell
+
+    (psamm-env) $ psamm-model fba R_CS
+
+The following is a sample of some output from the FBA command:
+
+.. code-block:: shell
+
+    (sample fba output waiting on loop commands to get real output)
+
+At the end of the output PSAMM will print the maximized flux of the designated
+objective function. The rest of the output is a list of the reaction IDs in the
+model along with their fluxes, fluxes with thermodynamic constraints imposed,
+and the reaction equations represented with the compound names.  This output is
+human readable because the reactions equations are represented with the full
+names of compound. It can be saved as a tab separated file that can be sorted
+and analyzed quickly allowing for easy analysis and comparison between FBA in
+different conditions.
+
+Flux balance analysis will be used throughout this tutorial as both a checking
+tool during model curation and an analysis tool. `PSAMM` allows the user to easily
+integrate analysis tools like this into the various steps during model
+development.
+
+6) Version Control with the YAML Format
+________________________________________
+
+The YAML format contains a logical division of the model information and
+allow for easier modification and interaction with the model. Moreover, the
+text-based representation of YAML files can enable the tracking of model
+modifications using version control systems. In this tutorial we will
+demonstrate the utilization of the Git version control system during model
+development to track the changes that has been added to an existing model.
+This feature will improve the documentation of model development process and
+improve collaborative annotations during model curation.
+
+A) Initate a Git Repository for the YAML Model
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Perhaps add a line here "If your unfamiliar with GitHub please follow their online tutorial for more clarification"